BrainCloudClient/src/BrainCloud/Unity/Scripts/HUD/HUDInfo.cs

```diff
@@ -22,7 +22,7 @@ public void OnHUDDraw()
 		{
 			GUILayout.BeginVertical ();
 			
-			BrainCloudClient bcc = BrainCloudWrapper.GetBC ();
+			BrainCloudClient bcc = BrainCloudLoginPF.BrainCloud.Client;
```

Comment:
But I do like this.. much b etter then 
BCLoginPF.s_brainCloud.Client lol

Reply:
For what it's worth, I have less of an issue with Capital B in code - since we've got coding / naming conventions to deal with.

So if it's consistant with how we're naming other variables, etc - i'm okay with the capital b.

For our doc examples, I'd prefer _brainCloud.service().method() as discussed. But I'm okay if internal to this class it is capital B.  (after all, all the classes are named with capital B as well).

@stevemrjones @bit-jonathanm